packages/cli/src/daemon-config.ts

```diff
@@ -260,6 +267,20 @@ export class DaemonDidResolversConfig {
 @jsonObject
 @toJson
 export class DaemonCeramicNodeConfig {
+  /**
+   * DID used to sign requests to CAS. This is automatically derived by the `did-seed`.
+   */
+  @jsonMember(String)
+  did: string
+
+  /**
+   * Seed used to sign requests to CAS.
+   * This is randomly generated if a config file is not found.
+   * When specifying in a config file, use the name 'did-seed'.
+   */
+  @jsonMember(String, { name: 'did-seed' })
```

Comment:
it's a good question. s3 actually be default will read creds from a file. and then otherwise from env var. kubo keeps the secret used to generate peer ids in the config file as well--this design is modeled after how kubo does it today.